### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Mahamadou-Nouridine**
In this PR I've done these tasks:
- Desktop version of the navbar
- Desktop version of the headline section
- Desktop version of works section
- Desktop version of about me section
- Desktop version of contact form

### CODE REVIEW

File: about.css [hunk 1]
```diff
@@ -256,4 +266,54 @@ button:disabled {
...
+
+  .project1-image {
+    padding-right: 24px;
+  }
+
+  .project1-card {
+    flex-direction: row;
+    align-items: start;
+    justify-content: space-between;
+  }
```
Comment (aamirkhan2478) [inline about.css:299]: - [ ] Your design looks very nice :clap: but if you see Figma's design the cards are not the same as your cards. Kindly stick to the Figma design and fix your issue. Also when you hover any card then the overlay will disappear and the button will be shown a comparison of  your design to the Figma design check below :point_down: 

**Your Design**
Screenshot from 2023-03-30 00-10-24

**Figma design**
Screenshot from 2023-03-29 23-58-57

- [ ] If you see all cards of Figma design you will see an overlay on all the cards except one card and that card you will style if someone is hovering over any of your cards the overlay will disappear and the button will be shown as I mentioned above :point_up: 
- [ ] Check below :point_down:  my video and forget about the design in this video you can just focus on when I hover over any of my cards the button is shown and the overlay is hidden :slightly_smiling_face: 

**[Video](https://user-images.githubusercontent.com/45226848/228644898-c8492779-4293-4033-9286-24438f55c9d7.webm)**

**_Note: You will implement these changes only for the desktop version, not for the mobile version_**

File: style.css [hunk 2]
```diff
@@ -189,3 +204,74 @@ footer {
...
+  }
+
+  .contact-form {
+    flex-direction: row;
+  }
+
+  .form-text {
+    text-align: start;
+    max-width: 455px;
+  }
```
Comment (aamirkhan2478) [inline style.css:272]: - [ ] Please stick to the Figma design your form heading is not the same as the Figma design so kindly fix it :+1: check below :point_down:  the comparison of your design and Figma design :slightly_smiling_face: 

**Your design**
Screenshot from 2023-03-30 00-26-37

**Figma design**
Screenshot from 2023-03-30 00-26-21

- [ ] So if you see your design the `text-align` of your `heading`  and Figma design `heading` is not the same. Also, please remove `three dots` after `something text` its not look professional so kindly fix all these changes. :+1:

File: index.html [hunk 3]
```diff
@@ -39,200 +39,230 @@ <h3 class="headline-title2">I'm A Software Developer</h3>
...
+                <a class="social" rel="noopener" target="_blank" href="https://twitter.com/Nouridine_Dino">
+                    <i class="fa-brands fa-twitter"></i>
+                </a>
+            </li>
+            <li>
+                <a class="social" rel="noopener" target="_blank" href="https://www.instagram.com/mahamadou_nouridine">
+                    <i class="fa-brands fa-instagram"></i>
+                </a>
+            </li>
+        </ul>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] As I mentioned above :point_up: please stick to the Figma design your social media icons are totally different from the Figma design. Check below :point_down: the comparison between your design and Figma design :+1: 

**Your design**
- Screenshot from 2023-03-30 00-40-27

- Screenshot from 2023-03-30 00-40-38

**Figma design** 
- Screenshot from 2023-03-30 00-41-06
- Screenshot from 2023-03-30 00-40-49

- [ ] So kindly use the same social media icons as mentioned in Figma design :slightly_smiling_face:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @Mahamadou-Nouridine , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters checked

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.css [hunk 4]
```diff
@@ -256,4 +272,66 @@ button:disabled {
...
```
Comment (aamirkhan2478) [inline about.css:299]: - [ ] Great job so far :clap: but as I mentioned in my previous review whenever hover over any card the `overlay` will disappear and pic will be shown clearly check below :point_down: the Figma and your design comparison :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-03-30 14-15-27

**Figma Design**
Screenshot from 2023-03-29 23-58-57

- [ ] If you check your design the picture is not shown clearly but if you check Figma design the Picture is clear whenever someone is hovering over any card so kindly fix this issue also :+1:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @Mahamadou-Nouridine , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters checked

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @Mahamadou-Nouridine, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters checked
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**